Rakefile

```
@@ -7,9 +7,17 @@ require "rubocop/rake_task"
 RuboCop::RakeTask.new
 
 BASE_DIR = File.dirname(__FILE__)
-codegen_rb_files = ["lib/protoboeuf/codegen.rb"]
-proto_files = Rake::FileList[File.join(BASE_DIR, "test/fixtures/*.proto")]
-rb_files = proto_files.pathmap("#{BASE_DIR}/test/fixtures/%n_pb.rb")
+LIB_DIR = File.expand_path(BASE_DIR, "lib")
+
+codegen_rb_files = ["lib/protoboeuf/codegen.rb", "lib/protoboeuf/autoloadergen.rb"]
+
+# Fixture protos we want to compile with protoc
+protoc_test_fixtures = Rake::FileList[File.join(BASE_DIR, "test/fixtures/*.proto")]
+protoc_test_fixtures_rb_files = protoc_test_fixtures.pathmap("#{BASE_DIR}/test/fixtures/%n_pb.rb")
```

Could this be simplified?
```suggestion
protoc_test_fixtures_rb_files = protoc_test_fixtures.pathmap("%X_pb.rb")
```